packages/react/src/components/Backdrop/Backdrop.stories.mdx

```diff
@@ -0,0 +1,65 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import {useArgs} from '@storybook/client-api';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import Backdrop from './Backdrop.tsx';
+import Button from '../Button/Button.tsx';
+
+export const meta = {
+  component: Backdrop,
+  title: StoryConfig.Backdrop.hierarchy,
+};
+
+<Meta title={meta.title} component={meta.component} />
+
+export const Template = args => {
+    const [{open}, updateArgs] = useArgs();
+      return (
```

Comment:
```suggestion
    return (
```

Reply:
Keeping empty lines like this is wrong in mdx. It breaks the mdx syntax.